app/tests/src/ingestion/test_pdf_postprocess.py

```diff
@@ -341,3 +364,42 @@ def test__add_link_markdown(caplog):
             ],
         ),
     ]
+
+
+def test_add_list_markdown(enriched_texts):
+    prev_enriched_text_val = None
+    first_list_level = _add_list_markdown(prev_enriched_text_val, enriched_texts[1])
+
+    second_list_level = _add_list_markdown(
+        enriched_texts[2],
+        EnrichedText(
+            text="• Sub nested item.",
+            type=TextType.LIST_ITEM,
+            headings=[Heading(title="Section 1", level=1)],
+            page_number=2,
+            stylings=None,
+        ),
+    )
+
+    assert first_list_level.text == "- First item."
+    assert second_list_level.text == "  - \n  -  Sub nested item."
```

Comment:
This desired result looks odd with a blank list item. Should this be:
```suggestion
    assert second_list_level.text == "    - Sub nested item."
```

This result is probably a result of `current_e_text.text.replace("\u2022", "\n  - ")`. We can pair on this to talk through what we want the result to look like and a good algorithm to achieve it.